discord/http.py

```diff
@@ -449,6 +449,7 @@ def send_message(
         message_reference: Optional[message.MessageReference] = None,
         stickers: Optional[List[sticker.StickerItem]] = None,
         components: Optional[List[components.Component]] = None,
+        flags: Optional[int] = 0,
```

Comment:
Why not use `None` if the parameter is optional
```suggestion
        flags: Optional[int] = None,
```